.pre-commit-config.yaml

```diff
@@ -0,0 +1,30 @@
+---
+repos:
+  - repo: https://github.com/adrienverge/yamllint
+    rev: v1.35.1
+    hooks:
+      - id: yamllint
+        entry: yamllint -c .yamllint.yaml .
+  - repo: https://github.com/lyz-code/yamlfix
+    rev: 1.17.0
+    hooks:
+      - id: yamlfix
+  - repo: https://github.com/shellcheck-py/shellcheck-py
```

Comment:
Let's find a param that we can pass to shellcheck to check also attached `sources` to bash files - https://github.com/koalaman/shellcheck

Reply:
Done